data/presets/presets/advertising/column.json

```diff
@@ -0,0 +1,14 @@
+{
+    "fields": [
+       "direction",
+       "lit"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "advertising": "column"
+    },
+    "name": "Column"
```

Comment:
Can you call it an "Advertising Column" - otherwise people will probably misuse the preset for architectural stuff.